lib/packages/ui/lib/src/utils/typedefs.dart

```diff
@@ -1,38 +1,111 @@
 import 'package:flutter/material.dart';
 import 'package:likeminds_chat_flutter_ui/likeminds_chat_flutter_ui.dart';
```

Comment:
We have to import the files inside `src` directory. Can't directly import the library inside itself. Remove from everywhere.

Reply:
done